install/source/index.markdown

```diff
@@ -15,36 +15,44 @@ MoveIt! is mainly supported on Linux, in particular:
 * Ubuntu 15.10, 16.04 / [ROS Kinetic](http://www.ros.org/wiki/kinetic/Installation/Ubuntu)
 * Ubuntu 14.04 / [ROS Indigo](http://www.ros.org/wiki/indigo/Installation/Ubuntu)
 
-### Prereq: Install ROS
+As of February 2019, [MoveIt for ROS 2.0](https://github.com/ros-planning/moveit2) development has begun but is not ready yet.
```

Comment:
Seems like an unrelated change?

Reply:
This PR updated the source install page. This is part of the source install page, and I don't think is very controversial or complex. So I see no harm in improving the documentation here, also.